### ISSUE TITLE
Redirection after successful activation

### ISSUE THREAD

**Issue by lingawarp** [QuodeIT/Quodeitv1.1#40]
even after successful activation of the round it is redirected to the activation page. It should be redirected to the correspoding page.

**Comment by tripples** [comment]
@lingawarp Is this issue appearing now ?

**Comment by lingawarp** [comment]
No, its not a problem actually. When you activate the round, after
successful activation it should be redirected to round page.

**Comment by tripples** [comment]
@lingawarp  Do we need any work on this ? I still not clear. If no work required to be done close issue.
